app/marketing/management/commands/expiration_start_work.py

```diff
@@ -60,20 +60,15 @@ def handle(self, *args, **options):
         for day in days:
             start_date = (timezone.now() - timezone.timedelta(days=(day+1)))
             end_date = (timezone.now() - timezone.timedelta(days=day))
-            interests_by_created_on = Interest.objects.select_related('profile').filter(
-                created__gte=start_date,
-                created__lt=end_date,
-                pending=False,
-            )
             interests_by_accepted_on = Interest.objects.select_related('profile').filter(
```

Comment:
I thought that the issue was to stop prodding people who hadn't been accepted to work on a bounty, have I misunderstood either the issue or what interest acceptance means?

Reply:
I thought that the issue was to stop prodding people who hadn't been accepted to work on a bounty, have I misunderstood either the issue or what interest acceptance means?

Reply:
Ah, okay, that makes sense. Should be an easy add, give me a minute and I'll have it pushed!

Reply:
This approach seems to eliminate the utility of this management command in regard to user prodding on permissionless bounties.  We need to account for permissionless (interest.accepted_on is None) **and** permissioned bounties where the interest was accepted (interest.pending is False, interest.accepted_on is some DT), not only where accepted_on is a dt.

Reply:
When you say permissionless bounties, do you mean bounties where the funder has specified that a user may start work without permission, eg. a collaborative bounty, or do you mean bounties where no worker has been given permission to start?

Reply:
@zachzundel Any bounty that doesn't have a permission associated with it.  For instance, a bounty_type `tranditional` with permission_type `permissionless`.  Collaborative bounties, contests, and traditional bounties without approval requirements.

Reply:
The only Interest objects that will have `accepted_on` dates set are in regard to traditional bounties with approval requirements.

Reply:
@zachzundel You'll probably want update the query to:

```python
.filter(
    Q(created__gte=start_date, created__lt=end_date, pending=False, acceptance_date__isnull=True) | \
    Q(acceptance_date__gte=start_date, acceptance_date__lt=end_date, pending=False)
).distinct('pk')
```

Then instead of `interest_day_0 = interest.acceptance_date` do something like:

`interest_day_0 = interest.acceptance_date or interest.created`